pkg/proctree/events_processing.go

```diff
@@ -0,0 +1,39 @@
+package proctree
+
+import (
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+// ProcessEvent update the process tree according to arriving event
+func (tree *ProcessTree) ProcessEvent(traceeEvent *trace.Event) error {
+	switch traceeEvent.EventName {
+	case "sched_process_fork":
+		return tree.processForkEvent(traceeEvent)
+	case "sched_process_exec":
+		return tree.processExecEvent(traceeEvent)
+	case "sched_process_exit":
+		return tree.processExitEvent(traceeEvent)
+	case "exit", "init_namespaces":
+		return nil
+	default:
```

Comment:
Discussing about having "high priority events" for internal processing (so we can stop trying to queue events and hold specific queues of the pipeline because of a parallel action): https://github.com/aquasecurity/tracee/issues/3086#issuecomment-1558482744

At the same time, the container enriching queueing seems to be working fine (although it took us some time to get there, and many stress tests).